frontend/src/components/AcmDataForm.tsx

```diff
@@ -585,94 +587,103 @@ export function AcmDataFormWizard(props: {
-                      <Button variant="secondary" onClick={onBack} isDisabled={isSubmitting}>
-                        {formData.backLabel}
-                      </Button>
-                    </ActionListItem>
-                  </ActionListGroup>
-                  <ActionListGroup>
-                    <ActionListItem>
-                      <Button variant="link" onClick={cancel} isDisabled={isSubmitting}>
-                        {formData.cancelLabel}
-                      </Button>
-                    </ActionListItem>
-                  </ActionListGroup>
-                </ActionList>
-              </ActionGroup>
-            </Stack>
-          )
-        }}
-      </WizardContextConsumer>
-    </WizardFooter>
-  )
+    return (
+      <Stack hasGutter style={{ width: '100%' }}>
+        {submitError && <Alert isInline variant="danger" title={submitError} />}
+        <ActionGroup>
+          <ActionList>
+            <ActionListGroup>
+              <ActionListItem>
+                <Button
+                  onClick={() => {
+                    if (!formHasErrors(t, formData)) {
+                      try {
+                        const result = formData.submit()
+                        if ((result as unknown) instanceof Promise) {
+                          setSubmitText(formData.submittingText)
+                          ;(result as unknown as Promise<void>).catch((err) => {
+                            if (err instanceof Error) setSubmitError(err.message)
+                            setSubmitText(formData.submitText)
+                          })
+                        }
+                      } catch (err) {
+                        if (err instanceof Error) setSubmitError(err.message)
+                      }
+                    }
+                  }}
+                  variant="primary"
+                  isDisabled={(showFormErrors && formHasErrors(t, formData)) || isSubmitting}
+                  isLoading={isSubmitting}
+                >
+                  {submitText}
+                </Button>
+              </ActionListItem>
+              <ActionListItem>
+                <Button variant="secondary" onClick={goToPrevStep} isDisabled={isSubmitting}>
+                  {formData.backLabel}
+                </Button>
+              </ActionListItem>
+            </ActionListGroup>
+            <ActionListGroup>
+              <ActionListItem>
+                <Button variant="link" onClick={cancel} isDisabled={isSubmitting}>
+                  {formData.cancelLabel}
+                </Button>
+              </ActionListItem>
+            </ActionListGroup>
+          </ActionList>
+        </ActionGroup>
+      </Stack>
+    )
+  }
 
   return (
     <Fragment>
       {isModalWizard ? (
-        <Wizard
-          titleId="create-credential-title"
-          descriptionId="create-credential-description"
-          title={formData.title}
-          description={formData.description}
```

Comment:
Title and description are missing from the in-wizard credential creation modal:
<img width="2557" alt="image" src="https://github.com/user-attachments/assets/84057005-00be-465a-8564-0af5b1238a5e" />